target_chains/cosmwasm/contracts/pyth/Cargo.toml

```diff
@@ -11,10 +11,12 @@ crate-type = ["cdylib", "rlib"]
 [features]
 # IMPORTANT: if you want to build for injective, enable the default feature below
 # default=["injective"]
+default=["osmosis"]
```

Comment:
do you want this to always be the default?

Reply:
oh yeah